src/Symfony/Component/Serializer/Normalizer/DenormalizableInterface.php

```diff
@@ -33,8 +33,6 @@ interface DenormalizableInterface
      * @param string|null                 $format       The format is optionally given to be able to denormalize
      *                                                  differently based on different input formats
      * @param array                       $context      Options for denormalizing
-     *
-     * @return object|object[]
```

Comment:
Not `@return` block would mean that this is a void method. I think, `@return mixed` is what you're looking for.

Reply:
No, I am actually looking for not returning anything, i.e. void. Look at How this method is used in the `CustomNormalizer`. The return value is not used at all.

Reply:
The absence of `@return` means `@return mixed`; `@return void` is different since it means it should not have its call result stored and doing so will produce an error if `: void` is enforced

Reply:
You are correct, my bad. I mixed up interfaces. An implementation of this interface should not return anything.